Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
 name = "zk-paillier"
-version = "0.1.1"
+version = "0.2.0"
 edition = "2018"
 
 authors = [
     "Gary <gary@kzencorp.com>",
     "Morten Dahl <mortendahlcs@gmail.com>",
-    "Omer <omer.shlomovits@gmail.com>"
+    "Omer <omer.shlomovits@gmail.com>",
+    "Oded Leiba <odedleiba5@gmail.com>"
```

Comment:
tbh, I do not think you contributed enough to this repo. 
However since this repo is not in crates.io I don't mind removing the authors section completely for now to make it more fair towards newer contributors. wdyt ?

Reply:
I'll remove my name